plugin/al/usdtransaction/AL/usd/transaction/__init__.py

```diff
@@ -19,7 +22,7 @@
 class ScopedTransaction(object):
 
     def __init__(self, stage, layer):
-        self.transaction = _AL_USDTransaction.Transaction(stage, layer)
+        self.transaction = Transaction(stage, layer)
```

Comment:
Removed the unneeded `_AL_USDTransaction` which causes a problem for the new case since we don't import it. This fixes the test "Python:AL_USDTransactionTests" which was failing.